go.mod

```diff
@@ -2,48 +2,46 @@ module github.com/red-hat-storage/ocs-client-operator
 
 go 1.18
 
-replace k8s.io/client-go => k8s.io/client-go v0.25.2
+replace (
+	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3 // required by rook v1.7
```

Comment:
we are not using Rook 1.7 even in ocs-operator why do we need this one?

Reply:
@iamniting suggested this to add, if not added build is failing.

Reply:
update the comment to point to rook 1.12 recent release?